modules/telco-core-red-hat-advanced-cluster-management.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/telco_core_ref_design_specs/telco-core-rds.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="telco-core-red-hat-advanced-cluster-management_{context}"]
+= Red Hat Advanced Cluster Management
+
+New in this release::
+* No reference design updates in this release
+
+Description::
+{rh-rhacm-first} provides Multi Cluster Engine (MCE) installation and ongoing zero-touch provisioning (ZTP) lifecycle management for deployed clusters.
+You manage cluster configuration and upgrades declaratively by applying `Policy` custom resources (CRs) to clusters during maintenance windows.
+
+You apply policies with the {rh-rhacm} policy controller as managed by {cgu-operator-full}.
+Configuration, upgrades, and cluster status are managed through the policy controller.
+
+When installing managed clusters, {rh-rhacm} applies labels and initial ignition configuration to individual nodes in support of custom disk partitioning, allocation of roles, and allocation to machine config pools.
+You define these configurations with `SiteConfig` or `ClusterInstance` CRs.
+
+Limits and requirements::
+
+* Hub cluster sizing is discussed in link:https://docs.redhat.com/en/documentation/red_hat_advanced_cluster_management_for_kubernetes/2.11/html-single/install/index#sizing-your-cluster[Sizing your cluster].
+
+* {rh-rhacm} scaling limits are described in link:https://docs.redhat.com/en/documentation/red_hat_advanced_cluster_management_for_kubernetes/2.11/html-single/install/index#performance-and-scalability[Performance and Scalability].
```

Comment:
Can't help but wonder if maintenance would be easier here if you made 2.11 an attribute. Just a thought, though, as I'm sure you've considered it.